draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -137,7 +137,7 @@ Issuer:
 :  An entity that creates SD-JWTs.
 
 Holder:
-:  An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term may refer to the actual user, the supporting hardware and software in their possession, or both.
+:  An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term usually refers to the actual user and the supporting hardware and software in their possession.
```

Comment:
I am not totally sure this change is needed or an improvement... ?

Reply:
agree with Brian..

Reply:
```suggestion
: An entity that received SD-JWTs from the Issuer and has control over them. In the context of this document, the term may refer to the actual user, the supporting hardware and software in their possession, or both.
```